privacyidea/lib/monitoringstats.py

```diff
@@ -27,19 +27,33 @@
 This module is tested in tests/test_lib_monitoringstats.py
 """
 import logging
-from dateutil.tz import tzlocal, tzutc
-import traceback
+from dateutil.tz import tzlocal
 from privacyidea.lib.log import log_with
-from privacyidea.lib.utils import convert_timestamp_to_utc
-from privacyidea.models import MonitoringStats
-from sqlalchemy import and_, distinct
-from privacyidea.lib.tokenclass import AUTH_DATE_FORMAT
+from privacyidea.lib.utils import convert_timestamp_to_utc, get_module_class
 import datetime
 
 log = logging.getLogger(__name__)
 
 
-def write_stats(stats_key, stats_value, timestamp=None, reset_values=False):
+@log_with(log, log_entry=False)
+def get_monitoring(config):
+    """
+    This wrapper function creates a new monitoring object based on the config
+    from the config file. The config file entry could look like this:
+
+        PI_MONITORING_MODULE = privacyidea.lib.monitoringmodule.sqlstats
+
+    Each monitoring module can have its own config values.
+
+    :param config: The config entries from the file config
+    :return: Audit Object
```

Comment:
isn't this a `Monitoring` object here?

Reply:
done in 905639b0a9974c9bbbaad243384e8539088fc1a1